tasks/src/unconditional-image-generation/about.md

```diff
@@ -0,0 +1,19 @@
+## About the Task
+
+Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
```

Comment:
```suggestion
Unconditional image generation is the task of generating images with no condition in any context (like a *prompt text* or *another image*). Once trained, the model will create images that resemble its training data distribution.
```
No need to write this given it's also on `data.ts` no?

Reply:
Oh sure! :)